Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -21,6 +21,47 @@
 # ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF
 # THE POSSIBILITY OF SUCH DAMAGE.
 
+# Web preferences are assigned a `status` which implies a default value and
+# controls the interfaces where the preference is declared.
+#
+# == Web Platform Features ==
+# unstable
+#   Feature in active development. Unfinished, no promise it is usable or safe.
+#   OFF by default.
+# testable
+#   Feature in active development. Stable enough for testing, but not ready to ship.
+#   OFF by default. Enabled in test infrastructure only.
+# preview
+#   Ready to be used in beta contexts like Safari Technology Preview.
+#   OFF by default. May be enabled automatically by clients.
+# stable
+#   Ready for general use, but should be able to be toggled to support
+#   debugging, A/B testing.
+#   ON by default.
```

Comment:
We should relax this a bit:

“ON by default, though some platforms may deviate from this typical behavior.” (or something along those lines)